src/lib/dynamodb.ts

```diff
@@ -14,13 +15,15 @@ export interface DynamoDBOptions {
 	accessKeyId?: string;
 	secretAccessKey?: string;
 	sessionToken?: string;
+	retries?: number | RetryOptions;
 }
 
 export class DynamoDB {
 
 	public raw?: AWS.DynamoDB;
 	public dynamodb?: AWS.DynamoDB.DocumentClient;
 	private options: DynamoDBOptions = {};
+	public retries?: number | RetryOptions;
```

Comment:
Maybe we just make this private and use a `get retries()`? That way, nobody else can change the value.